backend/s3/file.go

```diff
@@ -535,26 +536,35 @@ func waitUntilFileExists(file vfs.File, retries int) error {
 
 func (f *File) getReader() (io.ReadCloser, error) {
 	if f.reader == nil {
```

Comment:
GH does a poor job of showing that I just moved this down to be nested under 'else'